### DISCUSSION TITLE
Learning the Hanzi?

### DISCUSSION THREAD

**Discussion by Pu-Tai** [fabd/kanji-koohii#266] [2. Question & Answers] [answered]
Hi!

As I've understood it, there was once a twin-site to Kanji Koohii for the hanzi.

Is there any chance it will return? 

Best,

Pu-Tai

**Comment by fabd** [comment, marked as answer]
Hi there

I'm afraid not, as there was not even 10% of the activity seen on Koohii after I think almost two years of being online - the cost of maintaining the site, including a more complex codebase (able to run in two different "modes") could not be warranted.

That said, as stated in this [blog post](https://kanji.koohii.com/news/id/401) :

> (...) in practice all kanji and hanzi from RTK Volume 1/3, as well as RTH and RSH (Remembering the Hanzi books) are supported, both in Study pages and referencing in stories.

Since I plan to add JLPT indexes at some point (hence different Study page progress and lessons shown on dashboard) - I could in theory add RSH /RTH indexes in addition to the old/new RTK editions (edit: meaning it would be just one site, not two - so more reasonable implmentation).

... but in all likely hood, as I have to prioritize Koohii it is unlikely to happen.

**Reply by Pu-Tai** [reply]
Thank you for your reply!

Oh, that's too bad. I would use it even if it wasn't free but I understand if it's not a project you want to try.

**Reply by gary-host-laptop** [reply]
Is there a chance to bring it back to live if for example I pay for the cost of the hosting and so on? Or, how likely is for this one site, not two, thing to happen in a not so distant future? I assume this second option would be to add the RTH as an option much like how you can now select RTK editions, correct?

**Reply by fabd** [reply]
Thank you for your interest. At this point in time the only way I could see it come back is if someone talented enough with php and knowledgeable about Chinese/Hanzi made a PR that adds back the option - that would require having a single codebase just as it did in the past so I can deploy and maintain both versions online. The code used to have constants like CJ_HANZI

The code used to have a bunch of conditionals sprinkled in the code - there are still remnants such as here:
https://github.com/fabd/kanji-koohii/blob/3ee72dbcc9bc921ef0fb1e5465411ffca16b7218/src/apps/koohii/modules/account/templates/sequenceSuccess.php#L10

The more likely option would be to add additional sequences as the code is mostly ready for it - so you can pick RTH instead of RTK, since the db contains all the hanzi - but it wouldnt be a "chinese" version. Just the RTH/RSH sequences. Still would be nice. But also I don't have the time right now sadly - it requires checking a bunch of places in the code where there are assumptions about RTK frame number ranges.

Short answer : very unlikely.
